app/modules/backupTimer.js

```diff
@@ -0,0 +1,67 @@
+const db = require('better-sqlite3')('app.db');
+const fs = require('fs')
+
+const BACKUP_INTERVAL = 1000 * 60 * 60 * 24
+const MAX_BACKUPS = 100
```

Comment:
Good defaults, should be configurable by .env

Reply:
Adding these to .env.